mlir/include/mlir/Dialect/Bufferization/Transforms/Passes.h

```diff
@@ -164,8 +165,12 @@ struct BufferResultsToOutParamsOpts {
   bool addResultAttribute = false;
 
   /// If true, the pass eliminates the memref.alloc and memcpy if the returned
-  /// memref is allocated in the current function.
+  /// memref is static allocated in the current function.
```

Comment:
nit: "is allocated in the current function and has static shape"

The term "static allocation" sounds like `alloca`.